liquibase-core/src/main/java/liquibase/database/jvm/JdbcConnection.java

```diff
@@ -498,4 +513,10 @@ public boolean supportsBatchUpdates() throws DatabaseException {
             throw new DatabaseException("Asking the JDBC driver if it supports batched updates has failed.", e);
         }
     }
+   public static class Pair {
```

Comment:
Since Pair exists as an internal class in the Java world, you might rename this to something like **PatternPair** to distinguish it.